<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit_lastName">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Фамилия</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit_firstName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit_email">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>email</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit_phone">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Номер телефона</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Добавить новый контакт</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListView" name="listView">
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="btn_add">
     <property name="styleSheet">
      <string notr="true">QPushButton:hover {
	background-color: rgb(95, 95, 95);
}

QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(129, 129, 129);
	font: 75 14pt &quot;MS Shell Dlg 2&quot;;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: rgb(84, 84, 84);
}</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
